doc/src/build/json-rpc.md

```diff
@@ -130,32 +130,6 @@ curl --location --request POST $SUI_RPC_HOST \
               "id": 1 }' | json_pp
 ```
 
-#### Sign the transaction
-
-```shell
-sui keytool sign --address <owner_address> --data <tx_bytes>
-```
-The keytool creates a key and then returns the signature and public key information.
-
-#### Execute the transaction
-
-```shell
-curl --location --request POST $SUI_RPC_HOST \
---header 'Content-Type: application/json' \
---data-raw '{
-  "jsonrpc": "2.0",
-  "id": 1,
-  "method": "sui_executeTransaction",
-  "params": [ 
-    {{tx_bytes}},
-    {{sig_scheme}},
-    {{signature}},
-    {{pub_key}},
-    {{request_type}}
-  ]
-}'
-```
-
```

Comment:
https://github.com/MystenLabs/sui/pull/5530/files#diff-60958f8d2cfb073726cf0ad61ec376d78202f84662b5496596cc76edf991bafcR77 see the first time it comes up